addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/XxeScanRule.java

```diff
@@ -278,6 +289,38 @@ private void outOfBandFileInclusionAttack() {
         }
     }
 
+    private void outOfBandFileInclusionParameterEntityAttack() {
```

Comment:
The code should not be duplicated.

Reply:
Okay, will make changes to send parameter attack from the other OOB function itself.

Reply:
Updated